memory.cpp

```diff
@@ -141,14 +141,14 @@
     #else
         const int frames_to_skip = 7;
     #endif
-        int trace_size = backtrace(trace, stack_depth);
-        for (int i = frames_to_skip; i < trace_size; ++i)
-        {
-            Dl_info dl_info = {};
-            dladdr(trace[i], &dl_info);
-            if (dl_info.dli_fname && strstr(dl_info.dli_fname, "/IntelSEAPI."))
-                return true;
-        }
+        // int trace_size = backtrace(trace, stack_depth);
+        // for (int i = frames_to_skip; i < trace_size; ++i)
+        // {
+        //     Dl_info dl_info = {};
+        //     dladdr(trace[i], &dl_info);
+        //     if (dl_info.dli_fname && strstr(dl_info.dli_fname, "/IntelSEAPI."))
+        //         return true;
+        // }
```

Comment:
@araud are you sure you approve commenting out code? 😄

Reply:
I'd rather just remove code rather than comment it, personally. musl libc does not include a copy of libbacktrace (which provides `execinfo.h`) , unlike glibc, hence this functionality can't compile. libbacktrace support is a royal pain, too, when working with minimal build environments or LLVM and ideally shouldn't something that should be sneaking into production code.

Reply:
No, this patch has several changes, I have approved others, not this one.

Reply:
How are you approving just certain changes and not others? Can you point me to a view that shows that? Right now I see everything as approved.

Reply:
Maybe it doesn't really approves per change approval. Nevermind :) final approval will be in form of the submit of the changes from my side :)

Reply:
👍